src/directwrite/mod.rs

```diff
@@ -73,18 +72,18 @@ impl DirectWriteRasterizer {
             0.0,
             0.0,
         )
-        .map_err(Error::DirectWriteError)?;
+        .map_err(Error::from)?;
 
         let bounds = glyph_analysis
             .get_alpha_texture_bounds(dwrote::DWRITE_TEXTURE_CLEARTYPE_3x1)
-            .map_err(Error::DirectWriteError)?;
+            .map_err(Error::from)?;
 
         let buf = glyph_analysis
             .create_alpha_texture(dwrote::DWRITE_TEXTURE_CLEARTYPE_3x1, bounds)
-            .map_err(Error::DirectWriteError)?;
+            .map_err(Error::from)?;
 
         Ok(RasterizedGlyph {
-            c,
+            c: character,
```

Comment:
I'm not sure, but if you rename things anyways, it might be good to do it uniformly anywhere including the `RasterizedGlyph`?